### PR TITLE
enh(Resource-status/Filter): Change criterias to PopoverAutocomplete and add a filter summary

### PR DESCRIPTION
**Pull request by bdauria**
## Description

May-10-2021 14-56-24

**Fixes** # (issue)

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.10.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Filter/index.tsx [hunk 1]
```diff
@@ -128,48 +103,57 @@ const Filter = (): JSX.Element => {
...
+                  value={nextSearch || ''}
+                  onChange={prepareSearch}
+                  onKeyDown={requestSearchOnEnterKey}
+                />
+              </Grid>
+              <Grid item>
+                <Button
+                  color="primary"
+                  size="small"
+                  variant="text"
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/index.tsx:144]: Why the button is not contained anymore ?

**Review by Thebarda — COMMENTED** [review]
https://www.loom.com/share/950e34d456d34479b3f92d05f809c7cc

The "clear" button is missing. 
The selected values on monitoring servers autocomplete are not displayed in the summary

**Comment by bdauria** [comment]
> https://www.loom.com/share/950e34d456d34479b3f92d05f809c7cc
> 
> The "clear" button is missing.
> The selected values on monitoring servers autocomplete are not displayed in the summary

The clear button has been removed (same behaviour as setting the "All" filter)

File: www/front_src/src/Resources/Filter/index.tsx (hunk 1 above)
Comment (bdauria) [reply www/front_src/src/Resources/Filter/index.tsx]: > Why the button is not contained anymore ?

Fixed

**Review by jdelpierre — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]
Tested and LGTM

**Pull request merged**